kapost_deploy.gemspec

```diff
@@ -16,6 +16,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "rake", ">= 10.0"
   spec.add_dependency "platform-api", ">= 0.6.0"
   spec.add_dependency "slack-notify", ">= 0.4.1"
+  spec.add_dependency "honeybadger", "~> 2.0"
```

Comment:
...or make them optional. In other words, don't force the gem itself to require the gem dependency but build it as a dependency for the plugin only. I'll think about this more and submit a PR for it.